packages/lab/stories/combo-box-next/combo-box-next.doc.stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { ComboBoxNext } from "@salt-ds/lab";
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta title="Documentation/Lab/Combo Box Next" component={ComboBoxNext} />
+
+# Combo Box
+
+Combo Box helps users select an item from a large list of options without scrolling. The typeahead functionality makes this selection quicker, easier, and reduces errors.
+
+Users can see a list of available options when they click on the component and filter the list as they type. Once they’ve made their selection, it populates the field and the overlay list closes.
+
+<Canvas>
+  <Story id="lab-combo-box-next--default" />
+</Canvas>
+
+## How to use
+
+### When to use Combo Box
+
+You should use Combo Box when you have a list of 10 or more items, that don’t need to be permanently visible to the user like they would be in a filter panel or form.
```

Comment:
Phrasing Suggestion: 
Combo Box should be used when you have a list of 10 or more items, that don’t need to be permanently visible like they would be in a filter panel or form.